contracts/plugins/assets/Asset.sol

```diff
@@ -112,6 +112,14 @@ contract Asset is IAsset, VersionedAsset {
     /// @dev Should be general enough to not need to be overridden
     /// @return _low {UoA/tok} The lower end of the price estimate
     /// @return _high {UoA/tok} The upper end of the price estimate
+    /// @notice If the price feed is broken, _low will decay downwards and _high will decay upwards
+    ///     If the price feed is broken for more than `oracleTimeout + priceTimeout` seconds,
```

Comment:
Small nit I can make myself: Technically this isn't true due to the 2x part mentioned below, so really it should be "If `price()` is broken for more than..."